### ISSUE TITLE
No file data passed when using multipart/form-data file upload

### ISSUE THREAD

**Issue by FraserChapman** [jarrodek/ChromeRestClient#961]
Attempting to upload files via a POST request no file data is passed.

## Expected outcome
What should happen?

Would expect to see the boundary data and content-length for the attached files, e.g.
content-type: multipart/form-data; boundary=----WebKitFormBoundaryNnwvcXQcxXlK1Cng--
content-length: XX
------WebKitFormBoundaryNnwvcXQcxXlK1Cng--

## Actual outcome
What happened?

Get no file/boundary data, e.g.
content-type: multipart/form-data
content-length: 0

## Versions
App: 8.19.60.296-stable
Chrome: 56.0.2924.87
Platform: Mac OS X (10_12_3)

## Steps to reproduce
1. Create a POST request 
2. Attach a file or files to the request
3. Hit send and check the request headers

**Comment by FraserChapman** [comment]
Ah my bad, I see it is an issue with chrome: https://bugs.chromium.org/p/chromium/issues/detail?id=688100

**Comment by jarrodek** [comment]
Hi,

Thank you for issue report.
That's right. I reported this bug some time ago. However, I started working on a workaround. I think it will be beneficial for you because you'll have more control over multipart data. 
As soon as I finish it I'll make a new release. Unfortunately, it requires changes in the core of the application and the way how the app handing and sending data so It's not that simple to change it.
I'm doing my best to make it as soon as possible. But at the same time, I don't want to break the app.
Expect update rather sooner than later.

Thanks
